cfme/tests/configure/test_config_manual.py

```diff
@@ -122,6 +126,8 @@ def test_add_duplicate_subscription():
 
 
 @pytest.mark.manual
+@test_requirements.settings
+@test_requirements.multi_region
 @pytest.mark.tier(3)
 def test_add_bad_subscription():
     """
```

Comment:
_Optional_ `caseposneg:` should be `negative` for this test. By default, I believe it's positive.